index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
Remove this because you have deleted the `vite.svg` file too.

Reply:
done this